bubblewrap.c

```diff
@@ -131,11 +131,11 @@ lock_file_new (const char *path)
 
 
 static void
-usage (int ecode)
+usage (int ecode, FILE *out)
 {
-  fprintf (stderr, "usage: %s [OPTIONS...] COMMAND [ARGS...]\n\n", argv0);
+  fprintf (out, "usage: %s [OPTIONS...] COMMAND [ARGS...]\n\n", argv0);
 
-  fprintf (stderr,
+  fprintf (out,
```

Comment:
Sure, either way.  I generally separate my `usage()` output from a call to `exit`, so I like to make the output explicit, but both will work.